tls/s2n_resume.c

```diff
@@ -108,7 +143,7 @@ static int s2n_client_serialize_resumption_state(struct s2n_connection *conn, st
    }
 
     /* Serialize session state */
-    GUARD(s2n_serialize_resumption_state(conn, to));
+    GUARD(s2n_tls12_serialize_resumption_state(conn, to));
```

Comment:
This should technically be s2n_serialize_resumption_state, but updating it is covered in https://github.com/awslabs/s2n/issues/2490, so this is probably fine as-is.